Makefile

```diff
@@ -68,7 +68,7 @@ assets:
 
 promu:
 	@GOOS=$(shell uname -s | tr A-Z a-z) \
-	GOARCH=$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m))) \
```

Comment:
Why do we set GOOS and GOARCH explicitly here @sdurrheimer? What is the issue with a normal `$(GO) get -u github.com/prometheus/promu`?